odysseus/convertFtbQuests.ts

```diff
@@ -802,12 +802,13 @@ function convertReward(reward: QuestReward, rewardTables: (RewardTable & OrderIn
             if (rewardTable.rewards) {
                 let rewards: Record<string, HeraclesQuestReward> = {};
                 rewardTable.rewards.forEach((tableReward, i) => {
+                    const item = typeof tableReward.item === 'object' ? tableReward.item : {id: tableReward.item}
```

Comment:
Please put this shared logic in a function.

Reply:
extracted the conversion of an item reward using a new type in e9f5450

Reply:
Interesting approach, lgtm